src/utils.js

```diff
@@ -136,7 +136,30 @@ export const isPruneTraceOff = () =>
   );
 
 export const isSwitchedOff = () => {
-  return TracerGlobals.getTracerInputs().switchOff;
+  return TracerGlobals.getTracerInputs().switchOff || !isValidAlias();
+};
+
+export const getValidAliasesOrEmptyArray = () =>
```

Comment:
Why an empty array? If `process.env['VALID_ALIASES']` exists then you return a `string`, isn't it? I think that ENV variables are usually strings.

If anyway an array is returned then we can rename it:
```suggestion
export const getValidAliases = () =>
```